docs/theming.md

```diff
@@ -185,9 +185,170 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement. This migration guide will cover the typical migration path from Themer usage to
+Theming extensions.
```

Comment:
Tip: rearrange order of sentences to make it clearer what this article is about?  I would start with "this migration guide ..." and end with "themers will eventually be deprecated".
Also, do we assume the reader already knows the difference between themers and theming extensions?